src/PKSim.Core/Services/FileSelection.cs

```diff
@@ -31,6 +33,22 @@ public virtual string Description
          set => SetProperty(ref _description, value);
       }
 
+      public FileSelection AddSuffixToFileName(string suffix)
```

Comment:
In the spirit of trying to not mutate data...I am returning here a NEW instance with suffixed path and desription